ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -30,7 +30,7 @@ def initialize(opts = {})
 
       # Maintenance configuration
       @use_maintenance          = opts.fetch(:use_maintenance, true)
-      @maintenance_ip_whitelist = Array(opts.fetch(:maintenance_ip_whitelist, []))
+      @maintenance_ip_allowlist = Array(opts.fetch(:maintenance_ip_allowlist || :maintenance_ip_whitelist, []))
```

Comment:
I don't think this works with a simple `||`. You'll probably have to break it out into a function.

```
irb(main):009:0> opts = {:bar => "bar"}
=> {:bar=>"bar"}
irb(main):010:0> opts.fetch(:foo || :bar, [])
=> []
```